package.json

```diff
@@ -1,21 +1,19 @@
 {
-  "name": "typescript-action",
-  "description": "GitHub Actions TypeScript template",
-  "version": "0.0.0",
-  "author": "",
-  "private": true,
-  "homepage": "https://github.com/actions/typescript-action",
+  "name": "platformsh-action",
+  "description": "platform.sh action with deploy and clean environment feature",
+  "version": "1.0.0",
```

Comment:
How do we keep the version updated with the tag?

Reply:
I cannot think of an automated way. Probably, before releasing a new tag, we should change the version in package.json manually, push the changes, and then publish the new release. What are your thoughts? I am just wondering, is updating the version in package.json important?

Reply:
I don't think so. At least not as far as GitHub Actions is concerned. But in that case, let's not use it. As per [this link](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#version), `version` is optional as we don't plan to publish this on npm.